config/client_config.go

```diff
@@ -68,7 +68,7 @@ func NewDefaultClientConfig() *ClientConfig {
 		RsListHttpGetTimeout:             time.Duration(1000) * time.Millisecond, // 1s
 		RsListHttpGetRetryTimes:          3,
 		RsListHttpGetRetryInterval:       time.Duration(100) * time.Millisecond, // 100ms,
-		EnableRerouting:                  false,
+		EnableRerouting:                  true,
```

Comment:
Is there any problem with a server that doesn't have a Rerouting function?

Reply:
no problem, it doesn't work if a server that doesn't have a Rerouting function.